Backend/GroceryStore_Backend/Controllers/DepartmentController.cs

```diff
@@ -11,9 +11,9 @@ namespace GroceryStore_Backend.Controllers
     [Route("[controller]")]
     public class DepartmentController : Controller
     {
-        private readonly IGroceryStoreService _groceryStoreService;
+        private readonly IProductService _groceryStoreService;
 
-        public DepartmentController(IGroceryStoreService groceryStoreService)
+        public DepartmentController(IProductService groceryStoreService)
```

Comment:
I believe having separate service class for each entity would be a better design 
like DepartmentService.cs -> add, update, get, delete departments
ProductService.cs -> CRUD for Products